haystack/nodes/retriever/_embedding_encoder.py

```diff
@@ -374,7 +374,14 @@ def __init__(self, retriever: "EmbeddingRetriever"):
         self.model: str = next(
             (
                 m
-                for m in ["small", "medium", "large", "multilingual-22-12", "finance-sentiment"]
+                for m in [
+                    "small",
+                    "large",
+                    "multilingual-22-12",
+                    "embed-english-v2.0",
+                    "embed-english-light-v2.0",
+                    "embed-multilingual-v2.0",
+                ]
```

Comment:
nit: if this list is always the same, you can define it once in the `__init__.py` module of the `retriever` package so in the future we need to update it only in one place.